.github/workflows/wheels-Windows-MacOS.yml

```diff
@@ -0,0 +1,55 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Wheels for Windows and MacOS
+
+on:
+  push:
+    branches:
+      - master
+  release:
+    types:
+      - published
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [windows-2016, macOS-latest]
+        python-version: [3.5, 3.6, 3.7, 3.8] 
+
+    steps:
+    - uses: actions/checkout@v2
```

Comment:
This will probably need `git fetch --prune --unshallow` like `deploy.yml`:

https://github.com/ultrajson/ultrajson/blob/73463bce9ba335a907d8baf42e645e7ec414a29b/.github/workflows/deploy.yml#L19

The reason is so that setuptools_scm has the tags available, so can calculate a dev version number (eg. `2.0.4.dev7`), based on the "distance" since the last tag.